core/src/net/network_thread.rs

```diff
@@ -466,6 +467,7 @@ impl NetworkThread {
     fn send_tcp_message(&mut self, address: SocketAddr, data: DispatchData) {
         if let Some(channel_rc) = self.get_channel_by_address(&address) {
             let mut channel = channel_rc.borrow_mut();
+            let _ = self.get_channel_by_token(&channel.token);
```

Comment:
Comment that we are doing this to mark the channel as used. In fact, maybe let's have an extra function for this use-case. Maybe we can figure out a way to optimise this later (like `touched` function on the LRU or something).